conditional/templates/intro_evals.html

```diff
@@ -4,6 +4,19 @@
 {% endblock %}
 {% block body %}
 <div class="container main">
+<div class="row mobile-hide">
+    <div class="col-sm-10">
+    <h3 class="page-title">Intro Evaluations</h3>
+    </div>
+    <div class="col-sm-2">
```

Comment:
Could you re-align the indentation levels in this block for better readability?

Reply:
To clarify I mean the block under line 11, I'm not sure why GitHub's preview is showing the lines above.